humble/distribution.yaml

```diff
@@ -6299,6 +6281,24 @@ repositories:
       url: https://github.com/oKermorgant/slider_publisher.git
       version: ros2
     status: maintained
+  smacc2:
+    doc:
+      type: git
+      url: https://github.com/robosoft-ai/SMACC2.git
+      version: humble
+    release:
+      packages:
+      - smacc2
+      - smacc2_msgs
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros2-gbp/SMACC2-release.git
+      version: 2.3.5-1
```

Comment:
This version tag isn't available in the release repository https://github.com/ros2-gbp/SMACC2-release, CI won't pass until it is correctly bloomed.

Reply:
I already updated the SMACC2-release repo with new  versions. Think this is now fixed.

now I am getting the following error, what does that mean? It looks like it misses the package smacc2

FAILED test/test_build_caches.py::test_build_caches - RuntimeError: humble: missing package.xml file for package "smacc2"
humble: missing package.xml file for package "smacc2_msgs"

Reply:
It means that the version tag for `smacc2_msgs` is also not found.  It should follow the format described above, so `release/humble/smacc2_msgs/2.3.5-1`

I'm not seeing any tags in the smacc2-release repo that include the 2.3.5 version.

Are you using `bloom` to release this package?

Reply:
I see the versions correctly in the release repo:

https://github.com/robosoft-ai/SMACC2-release/tree/release/humble/smacc2_msgs/2.3.5-1
https://github.com/robosoft-ai/SMACC2-release/tree/release/humble/smacc2/2.3.5-1

Yes I am using bloom but it fails in the final PR step because I am getting problems due to SMACC2 is uppercase in the rosdistro.yaml file. 

My main goal is to fix that in order to enable the usage of bloom normally.

Reply:
You need to update the `url` field to reflect the new release location.  The release repo is currently here based on the rosdistro file: https://github.com/ros2-gbp/SMACC2-release.git

Reply:
I think this is already working

Reply:
Yep, this is in a lot better condition now.  Can you just update the description in the pull request body to match what this is doing?  I think you may have kept some of the other template sections in.

Reply:
would this update on the template be enough?